daemon/cluster/cluster.go

```diff
@@ -1764,7 +1764,7 @@ func (c *Cluster) populateNetworkID(ctx context.Context, client swarmapi.Control
 		apiNetwork, err := getNetwork(ctx, client, n.Target)
 		if err != nil {
 			if ln, _ := c.config.Backend.FindNetwork(n.Target); ln != nil && !ln.Info().Dynamic() {
-				err = fmt.Errorf("network %s is not eligible for docker services", ln.Name())
+				err = fmt.Errorf("The network %s cannot be used with services. Only networks scoped to the swarm can be used, such as those created with the overlay driver.", ln.Name())
```

Comment:
Having a lower case "the" doesn't work when there are multiple sentences, I think. Similarly, I added the `.` to form a complete second sentence, but I'm not sure what the Go conventions are there.

This is what the complete error looks like:

```
$ docker network create foo
$ docker service create --network foo nginx
Error response from daemon: The network foo cannot be used with services. Only networks scoped to the swarm can be used, such as those created with the overlay driver.
```